packages/opencensus-exporter-zipkin/test/test-zipkin.ts

```diff
@@ -109,6 +100,73 @@ describe('Zipkin Exporter', function() {
     });
   });
 
+  describe('translateSpan()', () => {
+    it('should translate traces to Zipkin format', () => {
+      const exporter = new ZipkinTraceExporter(zipkinOptions);
+      const tracer = new CoreTracer();
+      tracer.start(defaultConfig);
+
+      return tracer.startRootSpan({name: 'root-test'}, (rootSpan: RootSpan) => {
+        const span = rootSpan.startChildSpan('spanTest', SpanKind.CLIENT);
+        span.addAttribute('my-int-attribute', 100);
+        span.addAttribute('my-str-attribute', 'value');
+        span.addAttribute('my-bool-attribute', true);
+        span.setStatus(CanonicalCode.RESOURCE_EXHAUSTED, 'RESOURCE_EXHAUSTED');
+
+        span.addAnnotation('processing', {}, 1550213104708);
+        span.addMessageEvent(MessageEventType.SENT, '1', 1550213104708);
+        span.addMessageEvent(MessageEventType.RECEIVED, '2', 1550213104708);
+        span.addMessageEvent(MessageEventType.UNSPECIFIED, '3', 1550213104708);
+        span.addAnnotation('done', {}, 1550213104708);
+        span.end();
+        rootSpan.end();
+
+        const spanTraslated = exporter.translateSpan(rootSpan);
```

Comment:
Nit: `spanTranslated` not `spanTraslated` (missing an `n`)

Reply:
Done